frontstage/views/sign_in/sign_in.py

```diff
@@ -34,7 +35,9 @@ def login():
     form.username.data = form.username.data.strip()
     account_activated = request.args.get('account_activated', None)
 
-    if request.method == 'POST' and form.validate():
+    secure = Config.WTF_CSRF_ENABLED
```

Comment:
This should be loaded from the app config.

secure = app.config['WTF_CSRF_ENABLED']